beacon_node/network/src/metrics.rs

```diff
@@ -279,6 +279,11 @@ lazy_static! {
             "Gossipsub light_client_optimistic_update errors per error type",
             &["type"]
         );
+    pub static ref BEACON_PROCESSOR_SEND_ERROR: Result<IntCounter> =
```

Comment:
could also add a type field to differentiate errors originating from the Router struct vs the SyncNetworkContext struct